html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Document</title>
+  </head>
+  <body>
+  <h1>Coding Form!</h1>
+    <form action="www.codingform.com/why.php">
+      <p>
+        <label for="email">Email</label>
```

Comment:
This `label` isn't configured correctly, the value of the `for` attribute on a `label` should match the value of the `id` attribute on the form control element it is paired with.